cartridges/bm_imgix_pd/cartridge/experience/editors/imgix/imgixEditorTrigger.js

```diff
@@ -46,10 +46,10 @@ module.exports.init = function (editor) {
   // Create a configuration for a custom editor to be displayed in a modal breakout dialog (breakout editor)
   var breakoutEditorConfig = new HashMap();
   imgixApi["apiKey"] = currentSite.getCustomPreferenceValue(
-    "imgixPageDesignerAPIkey"
+    "ImgixapiKey"
```

Comment:
imgix should never be capitalized. It's weird I know 😆 
```suggestion
    "imgixApiKey"
```

Reply:
To clarify this suggestion, the name of the custom preference value should also be changed in the xml file